### PR TITLE
CML PR for exp_histgrad_person_split 3e2032b8

### PR DESCRIPTION
**Pull request by ghost**
Automated commits for https://github.com/Sensor-Based-Activity-Recognition/pipelines/commit/3e2032b815dad0cfdc53cf187c7dbfd8ae422943 created by CML.

### CODE REVIEW

**Comment by ghost** [comment]
# RUN: HistGradientBoostingClassifier, 2023-05-19 17-03-30

## Hyperparameters

| Path        | Param                                  | main                    | workspace                      |
|-------------|----------------------------------------|-------------------------|--------------------------------|
| params.yaml | dvclive.model                          | AdaBoostStumpClassifier | HistGradientBoostingClassifier |
| params.yaml | dvclive.model_hparams.learning_rate    | 0.5                     | 0.2                            |
| params.yaml | dvclive.model_hparams.max_bins         | -                       | 255                            |
| params.yaml | dvclive.model_hparams.max_iter         | -                       | 2000                           |
| params.yaml | dvclive.model_hparams.max_leaf_nodes   | -                       | 31                             |
| params.yaml | dvclive.model_hparams.min_samples_leaf | -                       | 20                             |
| params.yaml | dvclive.model_hparams.n_estimators     | 200                     | -                              |
| params.yaml | dvclive.model_hparams.random_state     | 42                      | -                              |
| params.yaml | dvclive.seed                           | 42                      | 41                             |
| params.yaml | train_test_split_ratio02.type          | measurement             | segment                        |

## Metrics

| Path                 | Metric          | main    | workspace   | Change   |
|----------------------|-----------------|---------|-------------|----------|
| dvclive/metrics.json | test.epoch.acc  | 0.70765 | 0.98525     | 0.2776   |
| dvclive/metrics.json | test.epoch.f1   | 0.66619 | 0.98363     | 0.31744  |
| dvclive/metrics.json | train.epoch.acc | 0.72471 | 1.0         | 0.27529  |
| dvclive/metrics.json | train.epoch.f1  | 0.68623 | 1.0         | 0.31377  |

## Plots

### Confusion Matrix

Confusion Matrix
